lib/Runtime/Library/WebAssemblyEnvironment.cpp

```diff
@@ -150,10 +150,7 @@ void WebAssemblyEnvironment::SetGlobalInternal(uint32 offset, T val)
 
 Wasm::WasmConstLitNode WebAssemblyEnvironment::GetGlobalValue(Wasm::WasmGlobal* global) const
 {
-    if (!global)
-    {
-        Js::Throw::InternalError();
-    }
+    AssertOrFailFast(global);
```

Comment:
nit: seems like a failfast is going too conservative, if global is null we'll just end up with a nullptr dereference

Reply:
The check is not very expensive (we've had it for so long) and I think I rather be conservative and also it documents the method that we don't allow nullptr for global